src/main/charts/jira/values.yaml

```diff
@@ -313,9 +313,9 @@ ingress:
    
   # -- The base path for the Ingress Resource. For example '/jira'. Based on a 
   # 'ingress.host' value of 'company.k8s.com' this would result in a URL of 
-  # 'company.k8s.com/jira'
+  # 'company.k8s.com/jira'. Default value is 'jira.service.contextpath'
   #
-  path: "/"
+  #path: '/'
```

Comment:
I don't think we comment out the values that are defaulted. You can leave it as `path: ` because providing the`path: /` example doesn't do more than the current default. Leaving the key there but without value is more consistent to rest of the keys.

When you change it to `path:`, please make sure it does still work with the `if` and ternary statements.

Reply:
done